src/gui/accountsettings.cpp

```diff
@@ -739,8 +739,8 @@ void AccountSettings::slotUpdateQuota(qint64 total, qint64 used)
         _ui->quotaProgressBar->setVisible(false);
         _ui->quotaInfoLabel->setToolTip(QString());
 
-        /* -1 means not computed; -2 means unknown; -3 means unlimited  (#3940)*/
-        if (total == 0 || total == -1) {
+        /* -1 means not computed; -2 means unknown; -3 means unlimited  (#owncloud/client/issues/3940)*/
+        if (quota == 0 || quota == -1) {
             _ui->quotaInfoLabel->setText(tr("Currently there is no storage usage information available."));
         } else {
```

Comment:
Another interesting thing: I tested with two accounts (different server versions each):
- `19.0.0.12`: Unlimited quota: Works fine, shows `2 GB in use`
- `18.0.2.2`: 200 GB quota: Only shows `Currently there is no storage usage information available.`

  The JSON:
  ```
    "quota": {
        "free": 13696878197,
        "used": 201051486603,
        "total": 214748364800,
        "relative": 93.62,
        "quota": 214748364800
    },
  ```

Reply:
Changing from `int` to `qint64` did the trick and `18.0.2.2` shows `187 GB of 200 GB in use` + progress bar :)